### PR TITLE
Added terraform for Vultr

### PR DESCRIPTION
**Pull request by aot9**
# Description

Added terraform configuration for Vultr.
Default config:
- Docker Ubuntu 20.04
- Seoul, South Korea
- 1 cpu 1 gb ram
- cronjob for periodic container restart

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)
- [x] Documentation update

## How Has This Been Tested?

- [x] Tested instance creation

## Screenshots

- [x] No screenshot

### CODE REVIEW

File: terraform/vultr/scripts/deploy.sh [hunk 1]
```diff
@@ -0,0 +1,8 @@
+#! /bin/bash
+
+apt install -y vnstat
+
+docker run -d -it --rm --name=d1000bn --pull always ghcr.io/arriven/db1000n-advanced
```
Comment (Arriven) [inline terraform/vultr/scripts/deploy.sh:5]: Typo in the name and can you use regular image with `-e ENABLE_PRIMITIVE=false`? The effect would be the same but I want to get rid of advanced image eventually as it takes a bit too long to be built

**Review by Arriven — COMMENTED** [review]
Looks good, left minor comment

File: terraform/vultr/scripts/deploy.sh [hunk 2]
```diff
@@ -0,0 +1,8 @@
+#! /bin/bash
+
+apt install -y vnstat
+
+docker run -d -it --rm --name=db1000n --pull always ghcr.io/arriven/db1000n
```
Comment (Arriven) [inline terraform/vultr/scripts/deploy.sh:5]:
```suggestion
docker run -d -it --rm --name=db1000n -e ENABLE_PRIMITIVE=false --pull always ghcr.io/arriven/db1000n
```

File: terraform/vultr/scripts/deploy.sh [hunk 3]
```diff
@@ -0,0 +1,8 @@
...
+
+sudo echo "0 */2 * * * docker kill db1000n || true && docker run -d -it --rm --name=db1000n --pull always ghcr.io/arriven/db1000n" >> cronjob
```
Comment (Arriven) [inline terraform/vultr/scripts/deploy.sh:7]:
```suggestion
sudo echo "0 */2 * * * docker kill db1000n || true && docker run -d -it --rm --name=db1000n -e ENABLE_PRIMITIVE=false --pull always ghcr.io/arriven/db1000n" >> cronjob
```

**Review by Arriven — COMMENTED** [review]
markdownlint is still complaining, so I also added suggestions on the docker command to mirror advanced image (that's what I meant on the first iteration)

File: terraform/vultr/scripts/deploy.sh (hunk 3 above)
Comment (aot9) [reply terraform/vultr/scripts/deploy.sh]: Done

**Review by Arriven — DISMISSED** [review]

**Review by Arriven — APPROVED** [review]

**Pull request merged**